backend-lib/src/main/java/cash/z/ecc/android/sdk/internal/model/CombinedWalletClientImpl.kt

```diff
@@ -124,7 +145,8 @@ class CombinedWalletClientImpl private constructor(
                 ServiceMode.DefaultTor,
                 is ServiceMode.Group ->
                     try {
-                        block(getOrCreate(serviceMode))
+                        val client = getOrCreate(serviceMode)
+                        block(client)
```

Comment:
This looks like it can be unaltered now, back to `block(getOrCreate(serviceMode))`.